axelor-account/src/main/java/com/axelor/apps/account/web/JournalController.java

```diff
@@ -85,4 +85,26 @@ public void toggleStatus(ActionRequest request, ActionResponse response) {
       TraceBackService.trace(response, e);
     }
   }
+
+  public void changeConfig(ActionRequest request, ActionResponse response) {
+    try {
+      Journal journal = request.getContext().asType(Journal.class);
+
+      if (journal.getCompany().getAccountConfig() == null) {
+        response.setValue("authorizeSimulatedMove", false);
+        response.setValue("allowAccountingDaybook", false);
+      } else {
+        if (journal.getCompany().getAccountConfig().getIsActivateSimulatedMove() == null
```

Comment:
You don't need to check if the IsActivateSimulatedMove is null, it's a boolean, not a Boolean, it can't be null